clients/pkg/logentry/stages/pack.go

```diff
@@ -38,7 +38,7 @@ func (w *Packed) UnmarshalJSON(data []byte) error {
 	w.Labels = map[string]string{}
 	for k, v := range *m {
 		// _entry key goes to the Entry field, everything else becomes a label
-		if k == logqlmodel.PackedEntryKey {
+		if k == logqllog.PackedEntryKey {
```

Comment:
It's imported as just `log` in `pack_test.go`?

Reply:
yeah I can't there because of `"github.com/go-kit/log"` I regret that package name but I couldn't find something better at that time :P

Reply:
I might follow up with a PR where I rename that package I think AI can find a good name :)